aws-functions/HttpTriggerAPIUsersUserId/lambda_function.py

```diff
@@ -0,0 +1,274 @@
+import logging
+import os
+import json
+import pymysql
+​
+​
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+​
+​
+def handler(event, context):
+    logger.info('Beginning function execution...')
+    logging.info(event)
```

Comment:
Is sticking to just logger better than using both logging and logger, since the level is explicitly set for logger?